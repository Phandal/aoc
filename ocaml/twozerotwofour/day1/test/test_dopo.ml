let l = [ "3"; ""; ""; ""; "4" ]

let test_empty =
  let t = Dopo.empty "" in
  Printf.printf "test_empty: %b\n" t

let test_not_empty =
  let t = Dopo.not_empty "hello" in
  Printf.printf "test_not_empty: %b\n" t

let test_filter =
  let nums = List.filter Dopo.not_empty l in
  Printf.printf "test_filter: %d\n" (List.length nums)

let () =
  test_empty;
  test_not_empty;
  test_filter
