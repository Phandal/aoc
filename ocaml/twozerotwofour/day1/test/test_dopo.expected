test_empty: true
test_not_empty: true
test_filter: 2
